webpack.config.js

```diff
@@ -29,46 +29,33 @@ exec(`pybabel compile -d ${localePath} -D sphinx`);
  * Paths for various assets (sources and destinations)
  */
 
-const vendorVersions = {fontAwesome: require("@fortawesome/fontawesome-free/package.json").version};
-
 const scriptPath = resolve(__dirname, "src/pydata_sphinx_theme/assets/scripts");
-const staticPath = resolve(__dirname,"src/pydata_sphinx_theme/theme/pydata_sphinx_theme/static");
+const staticPath = resolve(__dirname, "src/pydata_sphinx_theme/theme/pydata_sphinx_theme/static");
 const vendorPath = resolve(staticPath, "vendor");
-const faPath = {fontAwesome: resolve(vendorPath, "fontawesome", vendorVersions.fontAwesome)};
 
 /*******************************************************************************
  * functions to load the assets in the html head
  * the css, and js (preload/scripts) are digested for cache busting
  * the fonts are loaded from vendors
  */
 
-function stylesheet(css){ return `<link href="{{ pathto('_static/${css}', 1) }}?digest=${this.hash}" rel="stylesheet" />`;}
-function preload(js){ return `<link rel="preload" as="script" href="{{ pathto('_static/${js}', 1) }}?digest=${this.hash}" />`;}
-function script(js){ return `<script src="{{ pathto('_static/${js}', 1) }}?digest=${this.hash}"></script>`;}
-function font(woff2){ return `<link rel="preload" as="font" type="font/woff2" crossorigin href="{{ pathto('_static/${woff2}', 1) }}" />`;}
+function stylesheet(css) { return `<link href="{{ pathto('_static/${css}', 1) }}?digest=${this.hash}" rel="stylesheet" />`; }
+function preload(js) { return `<link rel="preload" as="script" href="{{ pathto('_static/${js}', 1) }}?digest=${this.hash}" />`; }
+function script(js) { return `<script defer src="{{ pathto('_static/${js}', 1) }}?digest=${this.hash}"></script>`; }
```

Comment:
Not for this PR< but if we compute hash, should use subresource integrity at some point ? 

https://developer.mozilla.org/en-US/docs/Web/Security/Subresource_Integrity

Reply:
Moved this to a new issue so we do not forget